src/Dumplie/Customer/Application/Query/Result/Cart.php

```diff
@@ -60,6 +83,6 @@ public function totalPrice() : float
      */
     public function currency() : string
     {
-        return $this->isEmpty() ? null : $this->items[0]->currency();
+        return $this->isEmpty() ? '' : $this->items[0]->currency();
```

Comment:
but for now, only way to set it is by putting product into cart
we can add optional constructor argument with currency